core/templates/services/image-upload-helper.service.ts

```diff
@@ -169,12 +169,24 @@ angular.module('oppia').factory('ImageUploadHelperService', [
       generateMathExpressionImageFilename: function(
           height, width, verticalPadding) {
         var date = new Date();
-        return 'mathImg_' +
-          _generateDateTimeStringForFilename() +
-          '_height_' + height +
-          '_width_' + width +
-          '_vertical_' + verticalPadding +
-          '.' + 'svg';
+        var filename = (
+          'mathImg_' +
+            _generateDateTimeStringForFilename() +
+            '_height_' + height +
+            '_width_' + width +
+            '_vertical_' + verticalPadding +
+            '.' + 'svg'
+        );
+        var filenameRegexString = (
```

Comment:
Define this in just one place -- probably assets/constants.ts if it's shared. Refer to that location from both the frontend and backend. Put a note above it to say: if this is changed, make sure that the names of existing math image files are handled properly as well.

Reply:
done